addons/project/static/src/components/project_list_renderer.js

```diff
@@ -0,0 +1,22 @@
+import { useService } from "@web/core/utils/hooks";
+import { ListRenderer } from '@web/views/list/list_renderer';
+
+import { useEffect } from "@odoo/owl";
+
+export class ProjectListRenderer extends ListRenderer {
```

Comment:
```suggestion
export class ProjectTaskListRenderer extends ListRenderer {
```

Can you also rename the file accordingly?

Thanks in advance! 🙂

Reply:
> Can you also rename the file accordingly?
> 
> Thanks in advance! 🙂


ProjectTaskListRenderer is already occupied, so can we rename it to ProjectSubTaskListRenderer?
[link](https://github.com/odoo/odoo/blame/master/addons/project/static/src/views/project_task_list/project_task_list_renderer.js#L7)

Reply:
TaskListRenderer, then. 🤷🏼‍♂️